pkg/cmd/cli/plugin/helpers.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright The Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	appsv1api "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const veleroLabel = "component=velero"
```

Comment:
We could make https://github.com/vmware-tanzu/velero/blob/main/pkg/install/resources.go#L57-L61 an exported function and use that label map here, to minimize duplication.

Reply:
Not sure if it reduces duplication...Here's what I came up with. Were you thinking of something different?

```
veleroLabel := install.Labels()

deployList, err := kubeClient.
	AppsV1().
	Deployments(namespace).
	List(ctx, metav1.ListOptions{
		LabelSelector: "component=" + veleroLabel["component"],
	})
```

The `LabelSelector` field takes a string while `install.Labels()` returns a map of labels.

Reply:
It reduces duplication because the same label key/value pair isn't defined in 2 files. We may run the risk of not updating both.

[The apimachinery library has a function to help with this](https://github.com/kubernetes/apimachinery/blob/master/pkg/labels/labels.go#L85), turning the map into a properly formatted string.

Reply:
Good call, and thanks for finding that library--exactly what I needed.

I have made the change you requested.